shared/fs/row/row.js

```diff
@@ -32,12 +34,37 @@ const HoverBox = isMobile
       },
     })
 
+const RowMeta = ({badgeCount, isNew, isIgnored, needsRekey}) => {
+  if (isIgnored || !(isNew || isIgnored || needsRekey || badgeCount)) {
+    return <Box />
+  }
+
+  return (
+    <Box style={{width: 0, display: 'flex'}}>
+      {needsRekey && (
+        <Box style={styleBadgeContainerRekey}>
+          <Meta title="rekey" style={badgeStyleRekey} />
+        </Box>
+      )}
+      {isNew && (
+        <Box style={styleBadgeContainerNew}>
+          <Meta title="new" style={badgeStyleNew} />
+        </Box>
+      )}
+      <Box style={styleBadgeContainer}>
+        {!!badgeCount && <Badge badgeNumber={badgeCount} badgeStyle={badgeStyleCount} />}
+      </Box>
+    </Box>
```

Comment:
Rekey needed can happen for ignored and new folders, but ignored and new are exclusive. Figured it was harmless to just have them non-exclusive in the content since they're exclusive in the data.